resources/compass/charts/cockpit/values.yaml

```diff
@@ -19,3 +19,9 @@ service:
 deployment:
   image:
     pullPolicy: IfNotPresent
+  # By default deploymentStrategy is set to rollingUpdate with maxSurge of 25% and maxUnavailable of 25% . you can change type to `Recreate` or can uncomment `rollingUpdate` specification and adjust them to your usage.
+  strategy: {}
+    # rollingUpdate:
+    #   maxSurge: 25%
+    #   maxUnavailable: 25%
+    # type: RollingUpdate
```

Comment:
Hmm I wonder if we need these two comments here.
I would suggest something more condensed, like:

```yaml
strategy: {} # Read more: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
```

If you like this idea, please change it in every chart.

Reply:
sure, changed